File: 02. C# Part2/03. Methods-Homework/07. ReverseNumber/ReverseNumber.cs
using System;

//Write a method that reverses the digits of given decimal number.

    class ReverseNumber
    {
        static void Main()
        {
            Console.WriteLine("Enter a integer number: ");
            int input = int.Parse(Console.ReadLine());
            int reversed = ReversedNumber(input);
            Console.WriteLine("The reversed number: {0}", reversed);
        }

        private static int ReversedNumber(int input)
        {
            int result = 0;
            while (input > 0)
            {
                result = result * 10 + input % 10;
                input /= 10;
            }
            return result;
        }
    }

File: 06. JavaScript Basics/06. Functions-Homework/06. Functions-Homework/02. ReverseNumber/ReverseNumber.js
//Write a function that reverses the digits of given decimal number.

function reverse(a)
{
    jsConsole.write("The reversed digit of the number " + a + " are: ");
    var result = 0;
    while(a){
        result = result * 10 + a % 10;
        a = Math.floor(a / 10);
    }
    jsConsole.writeLine(result);
}
reverse(256);